src/list/item/ListItem.tsx

```diff
@@ -5,7 +5,7 @@ import classNames from "classnames";
 
 import {KEYBOARD_EVENT_KEY} from "../../core/utils/keyboard/keyboardEventConstants";
 
-interface ListItemProps {
+export interface ListItemProps {
```

Comment:
We can export this in `index.ts` instead.

```ts
import {ListItemProps as ListItemComponentProps} from "...";
...
export type ListItemProps = ListItemComponentProps;
```

Reply:
Thank you 🙏